hushline/__init__.py

```diff
@@ -77,6 +77,14 @@ def inject_user() -> dict[str, Any]:
             return {"user": user}
         return {}
 
+    @app.context_processor
+    def inject_host() -> dict[str, HostOrganization]:
+        if (host_org := db.session.get(HostOrganization, 1)) is None:
```

Comment:
Altering the DB in a context processor feels like an abuse of this feature. Adding the row should happen in the migration, and optionally we could have some CLI command called like `db-doctor` that ensures any required state is present in the DB so that admins have a single command they could run if somehow the DB gets into an unexpected state.

Reply:
Instead of creating the `HostOrganization` here, can we add it via migration? Migrations aren't limited to DDL operations.

Reply:
that's fine, but can you open a follow up for this once it's merged so it doesn't get lost?